src/services/user.service.js

```diff
@@ -0,0 +1,47 @@
+const ApiError = require('../exeptions/api.error.js');
+const { User } = require('../models/user.js');
+const { uuidv4 } = require('uuid');
```

Comment:
The import statement for `uuidv4` is incorrect. It should be `const { v4: uuidv4 } = require('uuid');` to correctly import the `v4` function from the `uuid` package.